compiler/hash-token/keywords/hash.c

```diff
@@ -0,0 +1,174 @@
+/* ANSI-C code produced by gperf version 3.1 */
```

Comment:
Oh goodness.. Never thought I'd C the day...

Reply:
Removed `gperf` get we won't C the day just yet...